lib/oban/plugins/cron.ex

```diff
@@ -152,11 +152,13 @@ defmodule Oban.Plugins.Cron do
   end
 
   defp validate_opt!({:crontab, crontab}) do
-    unless is_list(crontab) and Enum.all?(crontab, &valid_crontab?/1) do
+    unless is_list(crontab) do
       raise ArgumentError,
             "expected :crontab to be a list of {expression, worker} or " <>
               "{expression, worker, options} tuples"
```

Comment:
The entry shape is validated elsewhere now:

```suggestion
      raise ArgumentError, "expected :crontab to be a list, got: #{inspect(crontab)}"
```